spec/requests/lobbies_spec.rb

```diff
@@ -226,4 +226,90 @@
       expect(Pusher).to have_received(:trigger).with(lobby.code, Lobby::PLAYER_JOIN, { id: Player.last.id })
     end
   end
+
+  describe 'start' do
+    let(:lobby) { FactoryBot.create(:lobby) }
+
+    before do
+      allow(Pusher).to receive(:trigger)
+    end
+
+    subject { patch start_api_v1_lobby_path(id: lobby.id) }
+
+    it 'responds with successful HTTP status' do
+      subject
+
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'responds with changed Lobby status to in_progress' do
```

Comment:
Another thing you might want to test is that the model itself is persisted with `in_progress` status 😄